src/map/clif.cpp

```diff
@@ -24795,6 +24798,67 @@ void clif_macro_reporter_status(map_session_data &sd, e_macro_report_status styp
 #endif
 }
 
+void clif_goldpc_info( struct map_session_data& sd ){
+#if PACKETVER_MAIN_NUM >= 20140508 || PACKETVER_RE_NUM >= 20140508 || defined(PACKETVER_ZERO)
+	const static int32 client_max_seconds = 3600;
+
+	if( battle_config.feature_goldpc_active ){
+		struct PACKET_ZC_GOLDPCCAFE_POINT p = {};
+
+		p.packetType = HEADER_ZC_GOLDPCCAFE_POINT;
+		p.active = true;
```

Comment:
to hide the icon on the battleground. thanks @KyoSasuke <3
```suggestion
		p.active = map_getmapflag(sd.bl.m, MF_BATTLEGROUND) ? false:true;
```

Reply:
> can you include not only for battleground. also for pvp and gvg. so you wont accidentally click while on battle

in this case, you must use the mapflag script. example:
```c
yourmappvp	mapflag	nodynamicnpc
```
this change that I made is to hide the icon that disturbs the BG scoreboard.
![image](https://user-images.githubusercontent.com/98105181/205710093-bdb5cf2a-a8b3-4a33-bc69-58370789979a.png)